helm-chart/flink-operator/templates/generate-cert.yaml

```diff
@@ -115,6 +115,33 @@ data:
         resources:
         - flinkclusters
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: flink-operator-cert-generator
+  namespace: {{ .Values.flinkOperatorNamespace }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: flink-operator-cert-manager
```

Comment:
Can we call this differently? We are not using cert-manager here.
Otherwise LGTM.